dts/arm/st/u5/stm32u5f9.dtsi

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2025 Charles Dias
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/u5/stm32u595.dtsi>
+
+/ {
+	soc {
+		compatible = "st,stm32u5f9", "st,stm32u5", "simple-bus";
```

Comment:
If you're adding a dts SoC definition, please also add the matching Kconfig definition.